src/components/site-search.js

```diff
@@ -412,17 +412,34 @@ LibrarySearchScopeOption.propTypes = {
 const ResultContent = ({ query, result }) => {
   return (
     <React.Fragment>
-      <p
-        data-title
-        css={{
-          ...TYPOGRAPHY.XS,
-          mark: {
-            background: `var(--color-maize-200)!important`,
-            fontWeight: '700'
-          }
-        }}
-      >
-        <HighlightText query={query} text={result.title} />
+      <p>
+        <span
+          data-title
+          css={{
+            ...TYPOGRAPHY.XS,
+            mark: {
+              background: `var(--color-maize-200)!important`,
+              fontWeight: '700'
+            }
+          }}
+        >
+          <HighlightText query={query} text={result.title} />
+        </span>
+        {result.tag && (
+          <span
+            css={{
+              color: 'var(--color-neutral-300)',
+              display: 'inline-block',
+              fontSize: '14px',
+              fontWeight: 'bold',
+              letterSpacing: '1.25px',
```

Comment:
```suggestion
              fontSize: '0.875rem',
              fontWeight: 'bold',
              letterSpacing: '0.0875em',
```
I understand Figma uses pixels for the typography so the measurements are an exact match, but for the sake of accessibility it's best to stick with `rem` or `em` for text measurements.